web/static/css/_grid-container.scss

```diff
@@ -0,0 +1,19 @@
+.grid-container {
+  @include margin(null auto);
+  @include padding(null $small-spacing);
+  max-width: 64em;
+
+  @include media($medium-screen-up) {
+    @include padding(null $large-spacing);
+  }
+
+  @include media($grid-breakpoint) {
+    display: grid;
```

Comment:
Do have need for a fallback or are you comfortable using CSS Grid only? Do you know what the layout looks like without CSS Grid?

Reply:
I'm comfortable with it, it's a website for designers. It's still readable without it, though, basically just a scaled up mobile view